scenarios/centos-9/multinode-ci.yml

```diff
@@ -20,3 +20,5 @@ cifmw_run_tests: true
 
 # Deploy EDPM in multinode scenario
 cifmw_deploy_edpm: true
+cifmw_tempest_tests_allowed:
+  - tempest.scenario.test_network_basic_ops.TestNetworkBasicOps.test_network_basic_ops
```

Comment:
if i recall correctly this also includes booting a vm is that working or do we need to skip it for now?

yeah found there: https://github.com/openstack/tempest/blob/2ce81e5185acc832da41709bc61379c66fe63adc/tempest/scenario/test_network_basic_ops.py#L37


and looking at your logs it says 'green' there https://logserver.rdoproject.org/02/502/aa73381d9f7aba488dddde72c3340f8a6f8ca19e/github-check/podified-multinode-edpm-deployment-crc/01175a3/controller/ci-framework-data/tests/tempest/stestr_results.html but I cannot see any more info. maybe we need to increase the logs verbosity?

[EDIT]: ok it looks like this is only running a small part of the test_network_basic_ops. I found an example from the mixed OS job, you can see it has 9 tests under this scenario https://logserver.rdoproject.org/openstack-component-compute/opendev.org/openstack/tripleo-ci/master/periodic-tripleo-ci-centos-9-8-multinode-mixed-os-compute-wallaby/b65303e/logs/undercloud/var/log/tempest/stestr_results.html.gz

so either we're skipping some of the tests somewhere else, or they cannot run and are just skipped automatically?

Reply:
Thank you @marios for review the pr. Below are the inline replies.

> if i recall correctly this also includes booting a vm is that working or do we need to skip it for now?
> 
> yeah found there: https://github.com/openstack/tempest/blob/2ce81e5185acc832da41709bc61379c66fe63adc/tempest/scenario/test_network_basic_ops.py#L37
> 
Yes correct. This test includes booting a volume.

> and looking at your logs it says 'green' there https://logserver.rdoproject.org/02/502/aa73381d9f7aba488dddde72c3340f8a6f8ca19e/github-check/podified-multinode-edpm-deployment-crc/01175a3/controller/ci-framework-data/tests/tempest/stestr_results.html but I cannot see any more info. maybe we need to increase the logs verbosity?
> 
> [EDIT]: ok it looks like this is only running a small part of the test_network_basic_ops. I found an example from the mixed OS job, you can see it has 9 tests under this scenario https://logserver.rdoproject.org/openstack-component-compute/opendev.org/openstack/tripleo-ci/master/periodic-tripleo-ci-centos-9-8-multinode-mixed-os-compute-wallaby/b65303e/logs/undercloud/var/log/tempest/stestr_results.html.gz
> 

Here is the detailed log for the tempest tests: https://logserver.rdoproject.org/02/502/aa73381d9f7aba488dddde72c3340f8a6f8ca19e/github-check/podified-multinode-edpm-deployment-crc/01175a3/controller/ci-framework-data/tests/tempest/tempest.log

> so either we're skipping some of the tests somewhere else, or they cannot run and are just skipped automatically?

Sounds good, let me run all network basic ops tests.